### PR TITLE
feat: implement player routes

### PR DESCRIPTION
**Pull request by falcao-g**

### CODE REVIEW

File: src/player/player.controller.js [hunk 1]
```diff
@@ -25,15 +26,31 @@ async function getPlayerById(playerId) {
 }
 
 function getDepositQuantity(req) {
-  const { quantity } = req.body;
+  const quantity = Number(req.query.quantity);
   if (!quantity) throw new ValidationError('Missing field: quantity');
-  if (!Number.isInteger(quantity)) throw new ValidationError('Quantity isn\'t a integer');
+  if (Number.isNaN(quantity)) throw new ValidationError('Quantity isn\'t a integer');
```
Comment (LKodex) [inline src/player/player.controller.js]: Number.isNaN doesn't guarantee that a number is a integer, the number may be a floating point number and won't be a NaN

So it's better to use !Number.isInteger

File: src/player/player.controller.js [hunk 2]
```diff
@@ -25,15 +26,31 @@ async function getPlayerById(playerId) {
...
 
-function addFundsTo(player, quantity) {
-  return database.player.incrementBalanceOfPlayer(player.uuid, quantity);
+async function addFundsTo(player, quantity) {
+  await database.player.incrementBalanceOfPlayer(player.uuid, quantity);
+  return `Added ${quantity} to balance succesfully`;
+}
+
+async function editPlayer(player, name, password) {
+  console.log(player);
```
Comment (LKodex) [inline src/player/player.controller.js]: Those console.log are for debug purpose? They should be here?

**Review by LKodex — CHANGES_REQUESTED** [review]

File: src/player/player.controller.js (hunk 1 above)
Comment (falcao-g) [reply src/player/player.controller.js]: fixed

File: src/player/player.controller.js (hunk 2 above)
Comment (falcao-g) [reply src/player/player.controller.js]: sorry, fixed

**Review by LKodex — APPROVED** [review]
Nice work, can be merged

**Pull request merged**